docs/guides/how_to_guides/creating_batches/how_to_load_a_spark_dataframe_as_a_batch.rst

```diff
@@ -3,16 +3,191 @@
+                    "dataset": insert_your_dataframe_here
+                    "data_asset_name": "optionally_insert_your_data_asset_name_here",
+                }
+
+            Then we get the Batch via:
+
+            .. code-block:: python
+
+                batch = context.get_batch(
+                    batch_kwargs=batch_kwargs,
+                    expectation_suite_name=suite
+                )
+
+
+        3. Check your data
+
+            You can check that the first few lines of your Batch are what you expect by running:
+
+            .. code-block:: python
+
+                batch.head()
+
+
+        Now that you have a :ref:`Batch <reference__core_concepts__datasources>`, you can use it to create :ref:`Expectations <expectations>` or validate the data.
+
+
+    .. tab-container:: tab1
+        :title: Show Docs for Experimental API (0.13)
+
+
+        What used to be called a “batch” in the old API was replaced with a :ref:`Validator <reference__core_concepts__validation>`. A Validator knows how to validate a particular batch of data on a particular :ref:`Execution Engine <reference__core_concepts>` against a particular :ref:`Expectation Suite <reference__core_concepts__expectations__expectation_suites>`. In interactive mode, the Validator can store and update an Expectation Suite while conducting Data Discovery or Exploratory Data Analysis.
+
+        You can read more about the core classes that make Great Expectations run in our :ref:`Core Concepts reference guide <reference__core_concepts>`.
+
+        .. admonition:: Prerequisites: This how-to guide assumes you have already:
+
+            - :ref:`Set up a working deployment of Great Expectations <tutorials__getting_started>`
+            - :ref:`Configured and loaded a DataContext <how_to_guides__configuring_data_contexts>`
+            - Identified a Spark DataFrame that you would like to use as the data to validate.
+
+        Steps
+        -----
+
+        0. Load or create a Data Context
+
+            The ``context`` referenced below can be loaded from disk or configured in code.
+
+            Load an on-disk Data Context via:
+
+            .. code-block:: python
+
+                import great_expectations as ge
+                context = ge.get_context()
+
+            Create an in-code Data Context using these instructions: :ref:`How to instantiate a Data Context without a yml file <how_to_guides__configuring_data_contexts__how_to_instantiate_a_data_context_without_a_yml_file>`
+
+
+        1. Configure a Datasource
+
+            Configure a :ref:`Datasource <reference__core_concepts__datasources>` using the :ref:`RuntimeDataConnector <reference__core_concepts__datasources>` to connect to your DataFrame. Since we are reading a Spark DataFrame, we use the ``SparkExecutionEngine``. You can use ``runtime_keys`` to define what data you are able to attach as additional metadata to your DataFrame using the ``partition_request`` parameter (shown in step 3).
```

Comment:
Maybe add a sentence like "You can use the below example as a template, and paste that into your great_expectations.yml" to make it a bit clearer how to use this.